test/e2e/app-dir/metadata-dynamic-routes/index.test.ts

```diff
@@ -326,16 +326,24 @@ createNextDescribe(
     it('should use localhost for local prod and fallback to deployment url when metadataBase is falsy', async () => {
       const $ = await next.render$('/metadata-base/unset')
       const twitterImage = $('meta[name="twitter:image"]').attr('content')
-
-      if (isNextDeploy) {
-        expect(twitterImage).toMatch(
-          /https:\/\/[\w-]+.vercel.app\/metadata-base\/unset\/twitter-image\.png/
+      const ogImages = $('meta[property="og:image"]')
+
+      ogImages.each((_, ogImage) => {
```

Comment:
Lets add an assertion that the length is greater than 0 because this test would pass right now if the length was 0